css/index.css

```diff
@@ -1,3 +1,451 @@
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  height: 100%;
+  font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+section {
+  position: relative;
+}
+
+h1 {
+  font-size: 60px;
+}
+
+.container {
+  margin: 0 auto;
+  width: 61%;
+  border: 1px solid YELLOW;
+}
+
+nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-content: center;
+  margin-top: 20px;
+  
+}
+nav hr {
+    display: none;
+}
+
+.menu a {
+  color: #777;
+  text-decoration: none;
+  padding-right: 40px;
+}
+
+.hero {
+  display: flex;
+  margin: 0 auto;
+  border-bottom: 2px solid #c9c9c9;
+  text-align: center;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.hero p {
+  text-align: center;
+}
+
+
+
+.hero img {
+  max-width: 300px;
+  margin: 40px 0;
+}
+
+.inner1 {
+  max-width: 50%;
+}
+
+.top-row {
+  display: flex;
+}
+
+.bottom-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.inner2 {
+  max-width: 30%;
+}
+
+.inner1 h2 {
+  margin-top: 20px;
+}
+
+.middle-img {
+  margin: 40px 0;
+}
+
+.bottom-row {
+  padding-bottom: 40px;
+  border-bottom: 2px solid #c9c9c9;
+}
+
+address {
+  margin-top: 40px;
+}
+
+.phone {
+  padding: 20px 0;
+}
+
+.copyright {
+  text-align: center;
+  margin: 80px 0;
+}
+
+/* Services Page-specific CSS */
+.services-header {
+	display: flex;
+	flex-direction: column;
+  margin: 20px 0;
+	padding-bottom: 30px;
+  border-bottom: 1px solid #c9c9c9;
+}
+
+.services-header h1 {
+	font-size: 28px;
+	padding-top: 20px;
+}
+
+.desc-section {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+	padding-bottom: 20px;
+  border-bottom: 1px solid #c9c9c9;
+}
+
+.service-desc {
+  max-width: 48%;
+  background-color: #F4F3F4;
+  border: 1px solid #c9c9c9;
+  padding: 20px;
+  margin: 20px 0;
+}
+
+.service-desc h2 {
+font-size: 20px;
+}
+
+button {
+  display: block;
+  margin-top: 15px;
+  padding: 8px 15px;
+  background-color: #fff;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.services-bottom-row {
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-start;
+	margin-top: 40px;
+}
+
+.services-bottom-row p {
+	padding-bottom: 20px;
+}
+
+/* Start of code */
+
+a {
+    
+    text-decoration: none;
+    color: gray;
+   
+}
+
+hr {
+    margin-bottom: 60px;
+    border: 1px black solid;
+    
+}
+
+.bottom {
+    margin-top: 60px;
+   
+}
+
+.logo {
+    display: flex;
+    flex-wrap: wrap;
+    width: 130px;
+    align-content: center;    
+    
+}
+
+
+.cta-text {
+    display:inline-block;
+    width: 300px;
+    text-align: center;
+    margin: 70px   ;
+    position:relative;
+    bottom: 60px;
+    font-size: 100px;
+
+}
+
+.circle {
+    display: inline-block;
+    margin-top: 150px
```

Comment:
Your 'margin-top: 150px' is likely the culprit that is causing troubles in your desktop view. Eliminate this and your desktop UI should improve drastically.